src/TextBoxGroup/text-box-group-style.scss

```diff
@@ -1,11 +1,13 @@
 //colors
-$red: #F03529;
-$red-darker: #BB000C;
-$blue: #41B0EE;
-$blue-darker: #007AC7;
-$gray-200: #e9e9e9;
-$gray-400: #cecece;
-$gray-500: #adadad;
+$red: #f03529;
+$grey6: #999999;
+$grey7: #b3b3b3;
+$grey8: #e6e6e6;
+$grey9: #f6f6f6;
```

Comment:
Try to find if these color names matches sebgroup/bootstrap colors under `_variables.scss` file

Reply:
resolved in commit https://github.com/sebgroup/react-components/pull/29/commits/2b668852b23d63790e2f37e7a82001a8673b8ad5